server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/IndicateHandCount.java

```diff
@@ -111,16 +103,16 @@ public synchronized String endpointBody(final Request the_request,
         serverError(the_response, "Could not select contests");
       } else {
         boolean hand_count = false;
-        final Set<County> hand_count_counties = new HashSet<>();
+        final Set<String> hand_count_contests = new HashSet<>();
```

Comment:
We might want to change [the message we return below](https://github.com/democracyworks/ColoradoRLA/pull/107/files#diff-9db23d940532ebce0b2b53d5e04c7d1fR122):

```java
ok(the_response, "Contests selected for hand count");
```